pipelines/skylab/optimus/Optimus.wdl

```diff
@@ -1,5 +1,5 @@
 version 1.0
-
+import "../../../FastqProcessing.wdl" as FastqProcessing
 import "../../../tasks/skylab/FastqToUBam.wdl" as FastqToUBam
```

Comment:
Are you going to take out the imports for the tasks FastqToUBam through SplitBamByCellBarcode?

Reply:
Yes, it is redundant now.

Reply:
Still planning to remove the redundant import?

Reply:
Can we remove the redundant import?